src/components/InfoPanel/infoPanel.module.scss

```diff
@@ -5,29 +5,31 @@
   background-color: white;
   display: flex;
   flex-direction: row;
-  justify-content: center;
-  align-items: center;
   position: absolute;
   border: 1px solid black;
   border-radius: 10px;
   z-index: 3000;
-  text-align: center;
-  bottom: -100px;
+  overflow-wrap: break-word;
+  bottom: -70px;
   right: auto;
   @media ($mobile) {
-    bottom: -100px;
+    bottom: -70px;
+    min-height: 120px;
```

Comment:
I think on mobile it still might look weird 
<img width="299" alt="image" src="https://github.com/michey85/IPAdress/assets/36237995/67fcef2d-99d1-4071-ab8b-4cfc9290e5e9">

I would try to limit ISP text by 3 lines. If there is more text it should be hidden by ellipsis. 
Try this https://www.geeksforgeeks.org/how-to-apply-an-ellipsis-to-multiline-text-in-css/

Reply:
And maybe make the font for InfoPanel a bit smaller for mobile.

Reply:
done